web/crux-ui/src/pages/products/[productId]/versions/[versionId]/deployments/[deploymentId]/instance/[instanceId].tsx

```diff
@@ -185,7 +187,7 @@ const InstanceDetailsPage = (props: InstanceDetailsPageProps) => {
 
           {getViewStateButtons()}
         </div>
-        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} initialBaseFilter="common" />}
+        {viewState === 'editor' && <ImageConfigFilters onChange={setFilters} initialBaseFilter="all" />}
```

Comment:
Why?

Reply:
TL;DR When I was testing as a **user** hidden fields in deployments were confusing.

I thought the we have bugs, but the fields that were in use were simply just hidden. On an image screen, it is perfectly fine to have hidden fields, but for a deployment it is obscure, everything that has some values, therefore will be deployed, should be visible. Templates have values that can get lost into the dark.
What do you think?

Reply:
Talked in person, okay for me.